Common/Basic components/CatalogCategories.cs

```diff
@@ -3,11 +3,11 @@
 
 namespace Infrastructure
 {
-    public class CatalogCategories : ComppenantBase
+    public class CatalogCategories : ComponentBase
     {
-        public Button Women => ParentElement.FindElements(By.CssSelector("#block_top_menu ul.sf-menu li a.sf-with-ul")).Select(element => new Button(Driver, element)).ToList()[0];
-        public Button Dresses => ParentElement.FindElements(By.CssSelector("#block_top_menu ul.sf-menu li a.sf-with-ul")).Select(element => new Button(Driver, element)).FirstOrDefault(element => element.ToString().ToLower() == "dresses");
-        public Button TShirts => ParentElement.FindElements(By.CssSelector("#block_top_menu ul.sf-menu li a.sf-with-ul")).Select(element => new Button(Driver, element)).FirstOrDefault(element => element.ToString().ToLower() == "t-shirts");
+        public Button Women => parentElement.FindElements(By.CssSelector("#block_top_menu ul.sf-menu li a.sf-with-ul")).Select(element => new Button(driver, element)).ToList()[0];
+        public Button Dresses => parentElement.FindElements(By.CssSelector("#block_top_menu ul.sf-menu li a.sf-with-ul")).Select(element => new Button(driver, element)).FirstOrDefault(element => element.ToString().ToLower() == "dresses");
+        public Button TShirts => parentElement.FindElements(By.CssSelector("#block_top_menu ul.sf-menu li a.sf-with-ul")).Select(element => new Button(driver, element)).FirstOrDefault(element => element.ToString().ToLower() == "t-shirts");
```

Comment:
Try to find the elements by their attribute tags. Look up how to get elements by their attributes css selector and try to find how you can fetch it better from the DOM